spec/presenters/step_by_step_presenter_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+require "gds_api/test_helpers/link_checker_api"
+
+RSpec.describe StepByStepPresenter do
+  include GdsApi::TestHelpers::LinkCheckerApi
+  include AuthenticationControllerHelpers
+  include GovukContentSchemaTestHelpers
+  include TimeOptionsHelper
+
+  before do
+    allow(Services.publishing_api).to receive(:lookup_content_id)
+    stub_user.name = "Firstname Lastname"
+  end
+
+  describe "#summary_list" do
+    let(:step_nav) { create(:step_by_step_page_with_steps) }
+    let(:default_summary) {
+      [
+        {
+          field: "Status",
+          value: "Draft"
+        },
+        {
+          field: "Last saved",
+          value: format_full_date_and_time(Time.zone.now)
+        },
+        {
+          field: "Created",
+          value: format_full_date_and_time(Time.zone.now)
+        }
+      ]
+    }
+
+    subject { described_class.new(step_nav).summary_list }
+
+    it "presents a list of key/value metadata" do
+      expect(subject).to eq(default_summary)
+    end
+
+    context "assigned_to is set" do
+      let(:step_nav) { create(:step_by_step_page_with_navigation_rules, assigned_to: stub_user.name) }
```

Comment:
The original `step_nav` could be re-used here rather than creating a new object.
For example: `step_nav.assigned_to = stub_user.name`. The assignment would have to be made inside the test, but as there's only one test, the context isn't really needed.